### PR TITLE
Removed generators in below files

### PR DESCRIPTION
**Pull request by mohitrpatil**
1. test_checkbyte.py
2. test_boyscout.py
3. test_relro_perm.py
4. test_accuracy.py
5. test_0div.py
6. test_keystone.py
7. test_veritesting.py

### CODE REVIEW

**Comment by ltfish** [comment]
Rebase your branch on the latest master.

**Review by twizmwazin — CHANGES_REQUESTED** [review]
Please rebase on master, and then we can take a look at why some of the tests are failing or timing out!

File: tests/test_0div.py [hunk 1]
```diff
@@ -41,4 +43,4 @@ def test_symbolic_0div(self):
 
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main()
```
Comment (twizmwazin) [inline tests/test_0div.py:46]: Missing ending newline

File: tests/test_keystone.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import os
 import logging
 import sys
+import unittest
 
 import unittest
```
Comment (twizmwazin) [inline tests/test_keystone.py:6]: One unittest import is enough!

File: tests/test_veritesting.py [hunk 3]
```diff
@@ -16,61 +17,82 @@
...
-    for arch in addresses_veritesting_a:
-        yield run_veritesting_a, arch
-
-def test_veritesting_b():
-    # Advanced stuff - it tests for the ability to inline simple functions
-    # as well as simple syscalls like read/write
-
-    for arch in addresses_veritesting_b:
-        yield run_veritesting_b, arch
+# <<<<<<< HEAD
```
Comment (twizmwazin) [inline tests/test_veritesting.py:20]: Looks like you left a rebase artifact in

File: tests/test_0div.py (hunk 1 above)
Comment (mohitrpatil) [reply tests/test_0div.py]: Resolved

File: tests/test_keystone.py (hunk 2 above)
Comment (mohitrpatil) [reply tests/test_keystone.py]: Resolved!

File: tests/test_veritesting.py (hunk 3 above)
Comment (mohitrpatil) [reply tests/test_veritesting.py]: Resolved! Removed the extra comment!

**Review by twizmwazin — COMMENTED** [review]
Please fix the lint issues and then this can be merged!

**Comment by twizmwazin** [comment]
Thanks!

**Pull request merged**